docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -559,17 +559,22 @@ Step 3 produces the following code values:
 .KS
    Symbol Length   Code
    ------ ------   ----
-   A       3        010
-   B       3        011
-   C       3        100
-   D       3        101
-   E       3        110
-   F       2         00
+   A       3       010
+   B       3       011
+   C       3       100
+   D       3       101
+   E       3       110
+   F       2       00
    G       4       1110
    H       4       1111
 .KE
 .fi
 
+The above prefix code contains codes that are parsed from left to right
+(with MSB on the left). All subsequent prefix codes in this
```

Comment:
On second thought, do we need this extra paragraph at all? I assume these refer to the variable length codes that are spelled out in the spec, but there we mention every time that the bits are parsed from right to left and we do not call them prefix codes, but variable length codes.